mappings/common/common.go

```diff
@@ -24,6 +42,19 @@ func MapToContract(m *engine.Model) *common.Model {
 	return &result
 }
 
+// MapOldContactToModel will map the spv-wallet-models contract to the common SPV Wallet Model
```

Comment:
Maybe let's add a bit more descriptive comment why it's named "MapOLD"... 
I am not a big fan of comments in code but I think that for short living things it's good.

Reply:
added Deprecation message